src/app/clusters/scenes-server/SceneHandlerImpl.cpp

```diff
@@ -58,7 +272,9 @@ DefaultSceneHandlerImpl::SerializeAdd(EndpointId endpoint, const ExtensionFieldS
     auto pair_iterator = extensionFieldSet.attributeValueList.begin();
     while (pair_iterator.Next())
     {
-        aVPairs[pairCount] = pair_iterator.GetValue();
+        AttributeValuePairType currentPair = pair_iterator.GetValue();
+        ReturnErrorOnFailure(ValidateAttributePath(endpoint, extensionFieldSet.clusterID, currentPair));
```

Comment:
So that will clamp to range, but for a nullable attribute, isn't it supposed to turn out-of-range to null instead of clamping?  Where is that implemented?  Or is that not relevant here?